docs/reference/src/main/asciidoc/ri-spi.asciidoc

```diff
@@ -637,9 +631,6 @@ Servlet deployment safely.
 
 NOTE: Weld only supports JSF 1.2 and above.
```

Comment:
Hm, I think JSF 1.2 is not relevant anymore. Weld 3 depends on JSF 2.2 API. I'm not sure how to update this statement. Given that our Tomcat example is using JSF 2.2 impl we should probably state that Weld 3 supports JSF 2.2+?